WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -1545,6 +1600,13 @@ - (void)showSettings
     [[QuickStartTourGuide find] visited:QuickStartTourElementBlogDetailNavigation];
 }
 
+-(void)showJetpackSettings
+{
+    [WPAnalytics trackEvent:WPAnalyticsEventJetpackSettingsViewed];
```

Comment:
Should we move this event to `viewDidLoad` in `JetpackSettingsViewController` since this view can be accessed in other areas of the app as well?

Reply:
Done in cbe18049e8229d0ff786cb82b676e90d0ed22039